README.md

```diff
@@ -14,7 +14,7 @@ Logs are stored as AWS CloudWatch metrics.
 	{
 	  "name": "Mailserver SMTP port check",
 	  "type": "port",
-	  "hostname": "mailserver.example.org",
+	  "url": "mailserver.example.org",
```

Comment:
I’ve changed `url` to `hostname` for check types that don’t actually require urls.

Reply:
True - but I think consistent config might be a bit easier to understand (from the user's point of view)

I made the change anyway